ImperatorToCK3.UnitTests/Mappers/Trait/TraitMapperTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using commonItems;
```

Comment:
There seems to be a mix of tabs and spaces in this file. Run the autoformatter again.

You might also check your editor settings about that. Or if you copy-pasted from somewhere, check the original source is what you ultimately want to use.

Reply:
Yeah, I copy-pasted the tests from C++ version to avoid making typos. The autoformatting is enabled, but I probably undid its changes after pasting for some reason. I set the indent to tabs now.